lib/exercism/user.rb

```diff
@@ -148,6 +148,16 @@ def invite_to_track_mentor(language)
     update_acls_for_track_mentor(language)
   end
 
+  def public_user_attributes
+    result = {}
+    result[:id] = self.id
```

Comment:
I don't think we expose the id for anything regarding users. Or rather, we might expose it, but when we look users up, it's always by email. Does the `id` from the database provide useful information?